Default Programmable Buffer Length for TX in rk3399's gmac is not suitable
for MTUs higher than 1498. The easiest solution would be to disable
TX offloading but it is not ideal as it disables hardware checksumming.

This patch sets txpbl to 0x4 which is good for the most popular MTU
value of 1500.

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index cede1ad81..02674bbf0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -288,6 +288,7 @@
 		resets = <&cru SRST_A_GMAC>;
 		reset-names = "stmmaceth";
 		rockchip,grf = <&grf>;
+		snps,txpbl = <0x4>;
 		status = "disabled";
 	};
 
